docs/content/guides/recipes/howto-author-recipes/snippets/environment.bicep

```diff
@@ -12,7 +12,7 @@ resource env 'Applications.Core/environments@2023-10-01-preview' = {
       'Applications.Datastores/redisCaches':{
         'redis-bicep': {
           templateKind: 'bicep'
-          templatePath: 'https://myregistry.azurecr.io/recipes/myrecipe:1.1.0'
+          templatePath: 'https://myregistry.ghcr.io/recipes/myrecipe:1.1.0'
```

Comment:
Need help checking that the target path for the command would still technically be valid here or if this requires `azurecr`.

Reply:
```suggestion
          templatePath: 'https://ghcr.io/USERNAME/recipes/myrecipe:1.1.0'
```